### DISCUSSION TITLE
Welcome to Instagram-DeepFake-Bot Discussions!

### DISCUSSION THREAD

**Discussion by dome272** [dome272/Instagram-DeepFake-Bot#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yezz123** [comment]
Thanks that is a great project and also helpful.
I'm an Enthusiastic about Deep learning and Machine Learning and soon i will be into this world ❤

**Reply by Ahmed1936l** [reply]
أهلاً بالجميع! 👋
أنا أحمد، مطور ويب مبتدئ ومهتم بالذكاء الاصطناعي والتصميم الجرافيكي. أحب تعلم كل جديد وتجربة أفكار مبتكرة. 🖌️💻
سعيد بانضمامي لهذا المجتمع، وأتطلع للتعرف عليكم ومشاركة الأفكار والإبداع معكم!
